### ISSUE TITLE
Airer light will be disconnected after low the airer

### ISSUE THREAD

**Issue by DZ8888** [al-one/hass-xiaomi-miot#517]
My Airer model is [beihao_airer](https://home.miot-spec.com/s/beihao.airer). Its function works well in HA through Xiaomi-miot
except one issue. the airer light will be unable to control after low the airer.
The light control is only available when airer is in highest position.
Is there anyone know how to fix this issue? Thanks!

**Comment by al-one** [comment]
Any error/warning [logs](https://my.home-assistant.io/redirect/logs).

**Comment by DZ8888** [comment]
Yes, there is an error in log as below.
Logger: custom_components.xiaomi_miot.light
Source: custom_components/xiaomi_miot/__init__.py:1242 
Integration: Xiaomi Miot Auto ([documentation](https://github.com/al-one/hass-xiaomi-miot), [issues](https://github.com/al-one/hass-xiaomi-miot/issues)) 
First occurred: 8:50:45 PM (1 occurrences) 
Last logged: 8:50:45 PM

Xiaomi Miot(beihao.airer.airer3): Got MiioException while fetching the state: Unable to parse message payload, mapping: {'light.on': {'siid': 3, 'piid': 1}, 'custom.left_time': {'siid': 5, 'piid': 1}, 'custom-5.left_time': {'siid': 5, 'piid': 2}, 'custom-5.left_time-3': {'siid': 5, 'piid': 3}, 'custom.air_dry': {'siid': 5, 'piid': 4}, 'airer.fault': {'siid': 2, 'piid': 1}, 'airer.motor_control': {'siid': 2, 'piid': 2}, 'airer.on': {'siid': 2, 'piid': 3}, 'airer.status': {'siid': 2, 'piid': 4}, 'airer.uv': {'siid': 2, 'piid': 6}, 'airer.dryer': {'siid': 2, 'piid': 7}}, max_properties: 6/11

**Comment by DZ8888** [comment]
By the way! the lowest position is too low to me, therefore I never low the airer to lowest position. Yesterday, I tried to low airer to lowest position. I found airer light control is available as highest position. As my understanding, there are 2 switch to detect the airer reach lowest and highest position. Form current symptom, it seems like something is going wrong while using stop button to stop airer in the area between lowest and highest position.
